account.go

```diff
@@ -72,6 +73,39 @@ func (a *Account) Balances() ([]horizon.Balance, error) {
 	return a.internal.Balances, nil
 }
 
+// IsMasterKeyActive returns whether the account's master key
+// has any chance of having power to sign transactions that will go through.
+func IsMasterKeyActive(accountID AddressStr) (bool, error) {
+	a := NewAccount(accountID)
+	err := a.load()
+	if err != nil {
+		if err == ErrAccountNotFound {
+			// Accounts with no entries have active master keys.
+			return true, nil
+		}
+		return false, err
+	}
+	minThreshold := int32(minBytes([]byte{a.internal.Thresholds.LowThreshold,
```

Comment:
can you add a comment what's going on here?

Reply:
Added at the top of the function